metapool/sample_sheet.py

```diff
@@ -11,109 +11,83 @@
 from metapool.plate import ErrorMessage, WarningMessage, PlateReplication
 
 
-_KL_SAMPLE_SHEET_SECTIONS = [
-    'Header', 'Reads', 'Settings', 'Data', 'Bioinformatics', 'Contact'
-]
-
-_KL_SAMPLE_SHEET_DATA_COLUMNS = [
-    'Sample_ID', 'Sample_Name', 'Sample_Plate', 'well_id_384', 'I7_Index_ID',
-    'index', 'I5_Index_ID', 'index2', 'Sample_Project', 'syndna_pool_number',
-    'Well_description'
-]
-
-_KL_SAMPLE_SHEET_COLUMN_ALTS = {'well_description': 'Well_description',
-                                'description': 'Well_description',
-                                'Description': 'Well_description',
-                                'sample_plate': 'Sample_Plate'}
-
-_KL_AMPLICON_REMAPPER = {
-    'sample sheet Sample_ID': 'Sample_ID',
-    'Sample': 'Sample_Name',
-    'Project Plate': 'Sample_Plate',
-    'Well': 'well_id_384',
-    'Name': 'I7_Index_ID',
-    'Golay Barcode': 'index',
-    'Project Name': 'Sample_Project',
-}
-
-_KL_METAGENOMIC_REMAPPER = {
-    'sample sheet Sample_ID': 'Sample_ID',
-    'Sample': 'Sample_Name',
-    'Project Plate': 'Sample_Plate',
-    'Well': 'well_id_384',
-    'i7 name': 'I7_Index_ID',
-    'i7 sequence': 'index',
-    'i5 name': 'I5_Index_ID',
-    'i5 sequence': 'index2',
-    'Project Name': 'Sample_Project',
-    'synDNA pool number': 'syndna_pool_number'
-}
-
-_KL_METATRANSCRIPTOMIC_REMAPPER = {
-    'sample sheet Sample_ID': 'Sample_ID',
-    'Sample': 'Sample_Name',
-    'Project Plate': 'Sample_Plate',
-    'Well': 'well_id_384',
-    'i7 name': 'I7_Index_ID',
-    'i7 sequence': 'index',
-    'i5 name': 'I5_Index_ID',
-    'i5 sequence': 'index2',
-    'Project Name': 'Sample_Project',
-    'synDNA pool number': 'syndna_pool_number'
-}
-
 _AMPLICON = 'TruSeq HT'
 _METAGENOMIC = 'Metagenomic'
 _METATRANSCRIPTOMIC = 'Metatranscriptomic'
-_ASSAYS = {_AMPLICON, _METAGENOMIC, _METATRANSCRIPTOMIC}
-
-_READS = {
-    'Read1': 151,
-    'Read2': 151
-}
-
-_SETTINGS = {
-    'ReverseComplement': '0',
-
-    # these are needed ever since we moved from bcl2fastq -> bclconvert
-    'MaskShortReads': '1',
-    'OverrideCycles': 'Y151;I8N2;I8N2;Y151'
-}
-
-
-_HEADER = {
-    'IEMFileVersion': '4',
-    'Investigator Name': 'Knight',
-    'Experiment Name': 'RKL_experiment',
-    'Date': None,
-    'Workflow': 'GenerateFASTQ',
-    'Application': 'FASTQ Only',
-    'Assay': None,
-    'Description': '',
-    'Chemistry': 'Default',
-}
-
-_BIOINFORMATICS_COLUMNS = {
-    'Sample_Project', 'QiitaID', 'BarcodesAreRC', 'ForwardAdapter',
-    'ReverseAdapter', 'HumanFiltering', 'contains_replicates',
-    'library_construction_protocol', 'experiment_design_description'
-}
+_STANDARD_SHEET_TYPE = 'standard_metag'
```

Comment:
I think these variables need to be as clear as possible and standard is not clear enough for me, what about using the name of the actual version? Something like `_METAGENOMICV100 = 'MetagenomicSampleSheetv100'` AND `_ABSQUANT = 'AbsQuantSampleSheetv10'`?